resources/views/admin/features/create_edit_feature.blade.php

```diff
@@ -191,18 +195,22 @@
                     </div>
                     <div class="col-md-6">
                         <div class="form-group">
+                            {!! Form::label('Rarity') !!}
+                            {!! Form::select('alt[rarity_id][]', $rarities, null, ['class' => 'form-control']) !!}
+                        </div>
+                    </div>
+                    <div class="form-group">
                             {!! Form::label('Display Mode') !!} {!! add_help("This controls how this alternate type's name will be displayed around the site. 'Name' refers to this type's name, whereas 'Parent Name' refers to the parent trait's name. Other values refer to this type's settings.") !!}
                             {!! Form::select('alt[display_mode][]', [
                                 0 => 'Name', 1 => 'Name (Species)',
                                 2 => 'Name (Subtype)', 3 => 'Parent Name (Name)',
                                 4 => 'Name Parent Name',
                             ], 0, ['class' => 'form-control']) !!}
-                        </div>
                     </div>
                     <div class="col-md-4">
                         <div class="form-group">
-                            {!! Form::label('Rarity') !!}
-                            {!! Form::select('alt[rarity_id][]', $rarities, null, ['class' => 'form-control']) !!}
+                            {!! Form::label('Trait Category (Optional)') !!}
+                            {!! Form::select('alt[feature_category_id][]', $categories, null, ['class' => 'form-control']) !!}
```

Comment:
I'm thinking it might be useful to have it autofill with the category of the parent trait at least at first since that will frequently (if not always) be the case (unlike with rarity, where it's frequently going to _differ_ from the parent trait)...

Reply:
I've opted to auto populate ALL fields, except Display. Not just to prevent errors (such as a missing rarity) but simply so that whenever an admin makes a new alt trait, they have the parent as base to _deviate from_.

Reply:
That makes sense to me!